modules/forum/src/main/PostRepo.scala

```diff
@@ -65,7 +65,7 @@ sealed abstract class PostRepo(troll: Boolean) {
     "text" -> post.text
   ))
 
-  def sortQuery = $sort.createdAsc
+  def sortQuery = $sort asc "number"
```

Comment:
this small change quietly invalidates the database indexes. Let's not change the value of createdAt - and use updatedAt instead.